initial-data-load/src/types.ts

```diff
@@ -38,9 +38,10 @@ export interface Metadata {
 }
 
 // API DATA FORMATS //
-export interface APIRegionAttribute {
+export interface APIFeatureAttribute {
   attributeId: string;
   featureId: string;
+  featureType: string;
```

Comment:
I would make it optional, as we might want to load data without specified featureType

Reply:
Yeah sure, fixed.